frog/imports/ui/GraphEditor/Graph.js

```diff
@@ -23,7 +23,7 @@ const scrollMouse = e => {
 };
 
 const mousemove = e => {
-  store.ui.socialMove(e.clientX, e.clientY);
+  store.ui.socialMove(e.clientX + 10, e.clientY - 75);
```

Comment:
I don't really like this change. It seems that for a lot of things the position is computed correctly. I'd rather identify where and how the positions are computed correctly and reuse it for the dragged connections if possible.

Also creating operators is broken (before the fix of this PR), but moving them works fine.

Reply:
I actually think this is OK - looking through, the only two places we use SocialCoords is in dropping operators, and dragging lines - everywhere else we do it with relative movement I think. Previously we were using offsetX/offsetY, which I guess calculates within the SVG, but is not supported on Firefox etc. 

Of course, if we ever make the graph location/size etc more flexible, we need to stop hard-coding this, but we're already hardcoding it in for example the Rename field, so I think it's OK for now. Anyway great to have this working on Firefox!